src/lib/Twig/Components/OverflowList.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\DesignSystemTwig\Twig\Components;
+
+use InvalidArgumentException;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\UX\TwigComponent\Attribute\AsTwigComponent;
+use Symfony\UX\TwigComponent\Attribute\ExposeInTemplate;
+use Symfony\UX\TwigComponent\Attribute\PreMount;
+
+#[AsTwigComponent('ibexa:overflow_list')]
+final class OverflowList
+{
+    /** @var list<array> */
+    public array $items = [];
+
+    /** @var list<string> */
```

Comment:
`list` type is PHPStan-specific. Generic phpDocumentor might not understand it (not relevant since it's not a contract, but as a principle)
```suggestion
    /** @phpstan-var list<array> */
    public array $items = [];

    /** @phpstan-var list<string> */
```